apps/frontend/src/components/ViewFormsList/MultiSelectBoxes.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from "react";
+import { Checkbox, Text, Box, Menu, MenuButton, MenuList, MenuItem } from "@chakra-ui/react";
+import { StatesDropdownValues } from "../../enums/DropdownValues";
+
+const states: string[] = ["CT", "ME", "MA", "RI", "VT", "Other"];
+
+const getStateName = (stateAbbr: string): string => {
+  switch (stateAbbr) {
+    case "CT":
+      return StatesDropdownValues.CT;
+    case "ME":
+      return StatesDropdownValues.ME;
+    case "MA":
+      return StatesDropdownValues.MA;
+    case "RI":
+      return StatesDropdownValues.RI;
+    case "VT":
+      return StatesDropdownValues.VT;
+    case "Other":
+      return StatesDropdownValues.OTHERSTATES;
+    default:
+      return stateAbbr;
+  }
+};
+
+export default function MultiSelect(): JSX.Element {
+  const [selectedStates, setSelectedStates] = useState<(keyof typeof StatesDropdownValues)[]>([]);
+  
+  const toggleState = (state: keyof typeof StatesDropdownValues) => {
+    setSelectedStates((prevSelectedStates) => {
+      if (prevSelectedStates.includes(state)) {
+        // Deselect the state
+        return prevSelectedStates.filter((s) => s !== state);
+      } else {
+        // Select the state
+        return [...prevSelectedStates, state];
+      }
+    });
+  };
+
+  return (
+    <div>
+      <Menu>
+        <MenuButton as={Box} cursor="pointer" borderWidth="1px" padding="10px" borderRadius="md" width="25%" margin="15px">
+          Filter by State    
+        </MenuButton>
+        <MenuList style={{ width: "200px"}}>
+          {states.map((stateAbbr, index) => {
```

Comment:
Rather than iterating over this new array of states that you defined in this file, you could iterate over the states enum that we already have.
So line 45 becomes
`Object.keys(StatesDropdownValues).map((stateAbbr, index) => {...`
That means you can delete the states array in this file and the getStateName method. Instead of getStateName you can use 
`StatesDropdownValues[stateKey]`

Reply:
line 45 where the filter by state label is?

Reply:
Sorry line 48